terraform/cloud_function/scheduled/variables.tf

```diff
@@ -1,8 +1,6 @@
-# must be defined in same file as where we define the variable with optional attrs
 terraform {
-  experiments = [module_variable_optional_attrs]
+  required_version = ">= 1.3.0"
```

Comment:
This is already in `main.tf` so I assume we need to set the constraint there? Not sure if terraform likes this in 2 different places.

Reply:
Thinking about it: even if Terraform doesn't mind, it gets pretty confusing for us if we add different version constraints throughout the code, so I guess I don't like it anyway :-)

Reply:
👍